app/views/templates/base.php

```diff
@@ -65,16 +78,13 @@
 <html lang="en" dir="ltr">
   <head>
     <title><?php if ($show_title == true) {
-    echo $page_title . ' | ';
+    print $page_title . ' | ';
 } ?><?=$title_productname?></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/style/transvision.css?<?php echo VERSION; ?>" type="text/css" media="all" />
-    <?php
-    foreach ($css_include as $css_file) {
-        echo "<link rel=\"stylesheet\" href=\"/style/{$css_file}?" . VERSION . "\" type=\"text/css\" media=\"all\" />\n";
-    }
-    ?>
+<?php foreach ($css_include as $css_file):?>
```

Comment:
I don't agree, php templating syntax is easier to read and less verbose since we don't have to output strings or escape quotes when we output html and this file *is* a template.